addons/hr_expense/static/src/views/attachment_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.AttachmentView" t-inherit-mode="extension" owl="1">
+       <xpath expr="//h3" position="replace">
+           <h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted d-flex justify-content-center">
```

Comment:
Since you're in master, I guess you could add a name or id in the parent `AttachmentView`, and only replace that `<t t-esc="state.thread.mainAttachment.filename"/>`, instead of having to copy again the following line
`<h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted d-flex justify-content-center">` ?

Reply:
I have replaced and added some css classes 
`<h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted float-end">` that why i have targeted the h3, what i could do is do another xpath but i think it will be less readable

Reply:
Oh right sorry, it looked very similar :p

Reply:
Oh right sorry, it looked very similar :p

Reply:
No probleeem

Reply:
Regarding the classes added, note that you are basically adding them to every single chatter due to the `t-inherit-mode="extension"`
I don't think that it is wanted; and if it was it should be done directly in `mail` anyways.